src/main/resources/jenkins/plugins/foldericon/CustomFolderIcon/config.jelly

```diff
@@ -4,18 +4,18 @@
 		<div id="icon-preview">
 			<j:choose>
 				<j:when test="${instance.foldericon != null &amp;&amp; instance.foldericon != ''}">
-					<img src="/userContent/customFolderIcons/${instance.foldericon}" height="32" width="32"/>
+					<img src="${app.rootUrl}userContent/customFolderIcons/${instance.foldericon}" height="32" width="32"/>
```

Comment:
Why `app.rootUrl` and not `rootURL` as elsewhere?

Reply:
found in some plugin sources as example. Not really sure what is auto-bind nowadays

Reply:
btw, am i correctly understand that there is no any other plugins beside folder plus with such functionality?

Reply:
To my knowledge this feature is only available in Folders Plus. Since there is little chance it could be included in the Folders Plugin I created this extension.
As for this PR I will test and merge it tomorrow and publish a new release. 
Thanks for your contribution, highly appreciated!

Reply:
@strangelookingnerd thanks